docs/configure

```diff
@@ -492,7 +487,6 @@ dnslookup:
   domains = ! +local_domains
   transport = remote_smtp
   ignore_target_hosts = 0.0.0.0 : 127.0.0.0/8
-  headers_remove = X-Spam-Report:X-Spam-Score
```

Comment:
go on then :)

I think I'd like to also explain how to compact `X-Spam-Report` to look like `X-Spam-Status`  somewhere in the config/manual, cuz I really love that bit. At the very least message preview is totally useless in that header.

BTW should we remove other headers usually added by SpamAssassin from received messages as well?